examples/tests/bare_bones/main.c

```diff
@@ -0,0 +1,29 @@
+// Do nothing. yield() immediately.
+
+// Define the `_start_` signature.
+void _start(void* app_start, void* mem_start, void* memory_len, void* app_heap_break);
+
+void _start(void* app_start __attribute__((unused)),
+            void* mem_start __attribute__((unused)),
+            void* memory_len __attribute__((unused)),
+            void* app_heap_break __attribute__((unused))) {
+  // Yield.
+#if defined(__thumb__)
+  __asm__ volatile (
+      "svc 0       \n"
```

Comment:
We might want to deterministically execute a yield-wait call here as well. (https://github.com/tock/tock/blob/a66504cd9bc339d85e857bec0afd9797e8d81297/kernel/src/syscall.rs#L42-L48)

I don't really know ARM ASM so don't want to make a suggestion. :)

Reply:
Adding a `movs r1, #1` before the `svc` should be portable across all cortex-m